fab/inventory/production

```diff
@@ -52,3 +52,18 @@ hqes3.internal.commcarehq.org elasticsearch_node_name=hqes3
 
 [shared_dir_host]
 hqdb0.internal.commcarehq.org
+
+[riakcs]
+hqriak0.internal.commcarehq.org
+hqriak1.internal.commcarehq.org
+hqriak2.internal.commcarehq.org
+hqriak3.internal.commcarehq.org
+hqriak4.internal.commcarehq.org
+
+[riakcs:vars]
+datavol_device=/dev/xvdb
```

Comment:
Cool! We don't use inventory params much, but I think this is actually a perfect use case for them: whether there's external block storage and where it lives is very much a part of the inventory and varies by machine (or in this case machine group). :+1: on the choice